debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/ConnectionContext.java

```diff
@@ -386,6 +397,17 @@ public void executeBlocking(String desc, BlockingConsumer<MongoClient> operation
                 return collections;
             });
         }
+
+        private boolean isRunning() {
```

Comment:
I see the discrepancy between the method name and the variable name. The method returns `true` if `MongoPrimary` is stopped.

Reply:
@jchipmunk Sigh, you are right, the correct code meant to be
`return !stop.get()`

Sorry for that :-(

Reply:
@jchipmunk Please check https://github.com/debezium/debezium/pull/636